generators/gulp/templates/gulpfile.js

```diff
@@ -42,7 +42,8 @@ gulp.task('test', ['pre-test'], function (cb) {
 
   gulp.src('test/**/*.js')
     .pipe(plumber())
-    .pipe(mocha({reporter: 'spec'}))
+    .pipe(mocha({reporter: 'spec'<% if (babel) { %>,
+      require: ['babel/register']<% } %>}))
```

Comment:
This is going to look a little strange:

```js
    .pipe(mocha({reporter: 'spec',
       require: ['babel/register']}))
```

it should be eighter all on one line, or all options on different lines.